examples/readme.md

```diff
@@ -0,0 +1,41 @@
+Logsink examples
+================
+
+Examples to create logsinks on Google Cloud Platform using the `logsink` module.
+
+Variables
+---------
+The Terraform variables for all the examples are defined in the `variables.tfvars` in the `examples/`
+directory. Replace the variables by your own before running the examples.
+
+Examples
+--------
+Examples are structured as follows:
+
+* ***base/*** creates a test folder identified by the variable `folder_name` and a test project identified by the variable `project_name`. This example **only** creates the structure required to run the other examples.
```

Comment:
I would call this setup and list it last.

Reply:
Yup.